bin/index.js

```diff
@@ -3,61 +3,83 @@
 const shell = require('shelljs')
 const request = require('superagent')
 
-const alias = process.argv[2].toLowerCase()
+const alias = process.argv[2]?.toLowerCase()
 const listingUrl = process.argv[3]
 
 if (!alias) {
   showUsage()
 } else if (alias === '--init') {
-  setupExtension(listingUrl)
+  const isOnCampus = process.argv[4] === '--on-campus'
```

Comment:
```suggestion
  const isOnCampus = process.env.ON_CAMPUS || process.argv[4] === '--on-campus'
```

Have those machines set an `ON_CAMPUS` environment variable and nobody needs to worry about providing an explicit flag while on campus, everyone else doesn't have to worry about their personal profile being nuked :)